helix-term/src/keymap/default.rs

```diff
@@ -274,6 +274,8 @@ pub fn default() -> HashMap<Mode, Keymap> {
             "r" => rename_symbol,
             "h" => select_references_to_symbol_under_cursor,
             "?" => command_palette,
+            "e" => reveal_current_file,
+            "E" => open_or_focus_explorer,
```

Comment:
![Screenshot_20230311_172724](https://user-images.githubusercontent.com/4687791/224476487-fac29fd8-80b0-4e4b-8348-07688aeaadb9.png)

Opening the tree explorer and then open a file is intuitive, but closing the tree explorer is not without looking through the docs.

The tree explorer seemed to be stuck on the side and I am not sure how to close it, I tried pressing `space` but it doesn't do anything within tree-explorer, maybe tree-explorer requires a different set of keys to handle it, but pressing `?` and then `q` still seemed acceptable to me. But the space seemed a bit wasted when in tree-explorer, maybe it should react like a normal file?

Also, when `<enter>` is pressed, should tree-explorer be closed instead?

Side note, I think we need to have a way to paginate the infobox on a small terminal given that it may overflow.

![Screenshot_20230311_172908](https://user-images.githubusercontent.com/4687791/224476546-5e294942-0108-491b-a543-b137c3f0f7a8.png)